esy.json

```diff
@@ -223,6 +224,80 @@
           "@opam/cmdliner": "^1.1.1"
         }
       }
-    }
+    },
+    "@opam/core": "InfiniteSwerve/core:core.opam#9c8f9f90ac74fe33aae13c7c040951a5ab78b064",
+    "@opam/core_bench": "janestreet/core_bench:core_bench.opam#48b0271934d742da433b109b41ccba53b5de4682",
```

Comment:
Should you also add it to `flake.nix`?

Reply:
We already have it in nix: https://github.com/marigold-dev/deku/blob/main/nix/deku.nix#L64